tests/integration/testdata/Makefile

```diff
@@ -0,0 +1,9 @@
+TEST_IMAGES ?= debian busybox hello-world
+CONTAINER_ENGINE := docker
```

Comment:
As this one is already defined in the top-level Makefile, you just need to `export` it in there.

Here, maybe, set it only if unset, e.g. `?=` -- in this case, if `make` will be run directly, this can still be set, otherwise, a value from the top-level Makefile will be used.

Need to be checked by changing the value in the top-level makefile and running `make .download-test-images`